source/develop/release-management/features/engine/metrics-store.html.md

```diff
@@ -39,20 +39,117 @@ We plan to continue adding additional logs and metrics as required, add pre-defi
 **Note:** Currently it should be installed on a new machine, separate from the engine. 
 It can be installed on a dedicated VM. 
 
-Please follow one of these options. Adding SSO is optional:
- 
-  * [Metrics Store setup on top of OpenShift with oVirt Engine SSO](https://www.ovirt.org/blog/2017/05/openshift-openId-integration-with-engine-sso/)
+Please follow the installation instructions:
+
+  * [Metrics Store setup on top of OpenShift](https://github.com/ViaQ/Main/blob/master/README-mux.md)
 
-  OR
+In oVirt 4.2 there will be an option to add SSO:
+  * [Metrics Store setup on top of OpenShift with oVirt Engine SSO](https://www.ovirt.org/blog/2017/05/openshift-openId-integration-with-engine-sso/)
 
-  * [Metrics Store setup on top of OpenShift without oVirt Engine SSO](https://github.com/ViaQ/Main/blob/master/README-mux.md)
 
 [![oVirt Metrics data flow](/images/wiki/oVirtMetricsDataFlow.jpg)](images/wiki/oVirtMetricsDataFlow.jpg)
 
 Once you have finished this step, you should have:
 
   * Kibana - <https://kibana.{hostname}>
   * OpenShift portal - <https://openshift.{hostname}>
+  
+  
+
+**Update mux pod resources**
+
+*mux* is short for multiplex, because it acts like a multiplexor or manifold,
+taking in connections from many collectors, and distributing them to a data
+store. 
+
```

Comment:
As discussed this should be explained in viaq documentation.
Here we only do the update.